packages/webapp/public/locales/en/translation.json

```diff
@@ -203,8 +203,16 @@
       "WHAT_TYPE_OF_IRRIGATION": "What type of irrigation?"
     },
     "MANAGE_CUSTOM_TASKS": "Manage custom tasks",
+    "MOVEMENT_VIEW": {
+      "MOVEMENT_PURPOSE_LABEL": "What's the purpose of this movement?",
+      "MOVEMENT_PURPOSE_PLACEHOLDER": "Select the purpose for this movement",
+      "OTHER_PURPOSE_EXPLANATION_LABEL": "Explanation",
+      "OTHER_PURPOSE_EXPLANATION_PLACEHOLDER": "Please describe the reason for this movement"
+    },
     "NEED_MANAGEMENT_PLAN": "You'll need an active or planned crop plan before you can schedule a harvest task or transplant task. Go to the crop catalogue to create a plan now.",
     "NO_MANAGEMENT_PLAN": "No eligible crop plans",
+    "NOTES_LABEL": "Any specific to add related to this task?",
```

Comment:
I see this came directly from Figma, but grammatically that's not quite right; it needs to be

"Anything specific to add..."

or

"Any specific notes to add ..."

Or even

"Any specifics to add..." 

(Just needs some noun or other 🙂) It's minor, but if we're going to be using it across all tasks now, I think it should be fixed. Maybe the version with "notes" is clearest and least redundant with the placeholder?

Reply:
I thought about this but then forgot to double check with Loïc! I agree, I'll change it to your first suggestion for now and double check with him just in case